README.md

```diff
@@ -119,6 +119,9 @@ These controls are accessible from any view:
 - `j`: Move song down in queue
 - `s`: Save the queue as a playlist
 - `S`: Shuffle the songs in the queue
+- `l`: Load a queue previously saved to the server
+
+When stmps exits, the queue is automatically recorded to the server, including the position in the song being played. There is a *single* queue per user that can be thusly saved. Because empty queues can not be stored on Subsonic servers, this queue is not automatically loaded; the `l` binding on the queue page will load the previous queue and seek to the last position in the top song.
```

Comment:
This is **a lie**. Seeking is what _should_ happen, but which doesn't, because of the failure of the code to actually seek without error.